.github/workflows/ci.yml

```diff
@@ -6,6 +6,13 @@ on:
     branches:
     - main
     - develop
+  workflow_dispatch:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
```

Comment:
I generally use true here, but this is fine too. We don't really need/use historical every-PR build status.